keypad/tests/test_rules.py

```diff
@@ -0,0 +1,86 @@
+"""File contains unittests for Rule class"""
```

Comment:
Nice, bare endre fra `../tests/..` til `../test/..` for at testene skal kjøre.
https://stackoverflow.com/a/6672873

Reply:
Done! La til test/__init__.py også!